pkg/cmd/roachtest/tests/admission_control_tpcc_overload.go

```diff
@@ -57,16 +59,23 @@ func (s tpccOLAPSpec) run(ctx context.Context, t test.Test, c cluster.Cluster) {
 	m := c.NewMonitor(ctx, c.CRDBNodes())
 	rampDuration := 2 * time.Minute
 	duration := 3 * time.Minute
+	labels := map[string]string{
```

Comment:
I'm not very familiar with the work going on here, but it seems these labels are consistent among the same workload, (i.e. all the TPCC tests set similar (same?) labels). Would it make more sense for this to be encoded at the workload layer instead of needing to specify in every single roachtest that needs it?

If that's not possible I think at least a `GetTPCCHistogramLabels` helper could be good to reduce the duplication here.